topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+
+![PDB ID: 4FR4]({% link topics/computational-chemistry/images/4FR4-STU.png %} "Crystal structure of the human serine/threonine-protein kinase in complex with staurosporine (PDB ID: 4FR4). Image generated using Maestro (Schrödinger LLC, NY).")
+
+Bioactive compounds often bind to several target proteins, thereby exhibiting polypharmacology. However, experimentally determining these interactions is laborious, and structure-based virtual screening of bioactive compounds could expedite drug discovery by prioritizing hits for experimental validation. The recently reported ePharmaLib ({% cite Moumbock2021 %}) dataset is a library of 15,148 e-pharmacophores modeled from solved structures of pharmaceutically relevant protein--ligand complexes of the screening Protein Data Bank (sc-PDB, {% cite Desaphy2014 %}). ePharmaLib can be used for target fishing of phenotypic hits, side effect predictions, drug repurposing, and scaffold hopping.
+
+![STU]({% link topics/computational-chemistry/images/STU.png %} "Overlay of staurosporine onto its key pharmacophoric features (PDB ID: 4FR4). Image generated using Maestro (Schrödinger LLC, NY).")
+
+In this tutorial, you will perform pharmacophore-based target prediction of a bioactive ligand known as staurosporine (Figure 2) with the ePharmaLib subset representing human protein targets (5,010 pharmacophore models) and the open-source pharmacophore alignment program [Align-it](https://anaconda.org/bioconda/align_it), formerly known as PHARAO ({% cite Taminau2008 %}).
+
```

Comment:
This is a nice introduction, but I feel a dedicated section on pharmacophores would be useful - maybe you could show a simple example file and explain the meaning of the different rows and columns? I think together with Figure 2 this would illustrate the concept very well for newcomers.

Reply:
In the tutorial, after uploading the ePharmaLib dataset, I suggested to the users to view its contents by pressing the **eye icon** (View data).